src/components/Searchbar/Searchbar.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useState } from "react";
+import {
+  InputGroup,
+  InputLeftElement,
+  Input,
+  Flex,
+  Button,
+  IconButton,
+  useMediaQuery,
+} from "@chakra-ui/react";
+import { SearchIcon } from "@chakra-ui/icons";
+import FilterMenu from "../FilterMenu";
+
+function Searchbar() {
+  const searchRef = useRef();
+
+  const [isLargerThanSmall] = useMediaQuery("(min-width: 30em)");
+  const [selectedFilters, setSelectedFilters] = useState([]);
+
+  const handleSearch = () => {
+    console.log(selectedFilters)
+  };
+
+  const handleFilterClick = (filters) => {
+    setSelectedFilters(filters);
+  };
+
+  return (
+      <Flex
+        alignItems="center"
+        justifyContent="space-between"
+        width="100%"
+        mx="auto"
+        mb="4"
+        px="4"
+      >
+        <InputGroup flex="1">
+          {isLargerThanSmall ? (
```

Comment:
You should try to move JS rendering logic like this into other functions to keep the main JSX templating cleaner.

Reply:
Moved it innto SearchInput component